guides/source/getting_started.md

```diff
@@ -72,92 +72,63 @@ By following along with this guide, you'll create a Rails project called
 `blog`, a (very) simple weblog. Before you can start building the application,
 you need to make sure that you have Rails itself installed.
 
+This guide makes use of Dev Containers, which allow you to work with your Rails
+application in a container, without needing to install Rails or its dependencies
+directly on your machine. This is the fastest way to get started with Rails.
+
+If you would like to install Rails directly on your machine, you can see the
+[Installing Rails guide](installing-rails.html)
+
 NOTE: The examples below use `$` to represent your terminal prompt in a UNIX-like OS,
 though it may have been customized to appear differently. If you are using Windows,
 your prompt will look something like `C:\source_code>`.
```

Comment:
With the re-arrangement of the sections, this note feels a little out of place before the installation section, maybe we could move it to after the intro paragraph in "Creating a Blog Application"?